sdk/react/widgets/CastPlayPauseButtons.js

```diff
@@ -0,0 +1,338 @@
+      fontFamily: icons[name].fontFamily,
+    };
+    const sizeStyle = {
+      width: buttonWidth,
+      height: buttonHeight,
+    };
+    const opacity = { opacity: skipButtons.animationOpacity };
+    const animate = { transform: [{ scale: skipButtons.animationScale }] };
+
+    let color = 'gray';
+    if (active) {
+      color = !!buttonColor ? buttonColor : 'white';
+    }
+
+    const isForward = name === NEXT;
+
+    return (
+      <SwitchButton
+        visible={active}
+        isForward={isForward}
+        sizeStyle={sizeStyle}
+        disabled={!active}
+        onSwitch={forward => this.onSwitchPressed(forward)}
+        icon={icons[name].icon}
+        fontStyle={fontStyle}
+        opacity={opacity}
+        animate={animate}
+        buttonColor={{ color }}
+      />
+    );
+  }
+
+  // Gets the play button based on the current config settings
+  render() {
+    const {
+      frameHeight, buttonHeight, frameWidth, buttonWidth, showButton, config,
+    } = this.props;
+    const {
+      previousVideo, nextVideo, skipBackward, skipForward,
+    } = config.castControls.buttons;
+    const seekButtonScale = 0.5;
+    const playPauseButton = this.renderPlayPauseButton();
+    const previousButton = this.renderSwitchButton(PREVIOUS, seekButtonScale, previousVideo.enabled);
+    const nextButton = this.renderSwitchButton(NEXT, seekButtonScale, nextVideo.enabled);
+    const backwardButton = this.renderSeekButton(BACKWARD, seekButtonScale, skipBackward.enabled);
+    const forwardButton = this.renderSeekButton(FORWARD, seekButtonScale, skipForward.enabled);
+
+    const containerStyle = {
+      flexDirection: 'row',
+      flex: 0,
+      justifyContent: 'space-between',
+      alignItems: 'center',
+    };
+
+    const heightScaleFactor = 2;
+    let widthScaleFactor = 2;
+    if (previousVideo.enabled) {
+      widthScaleFactor += 1;
+    }
+    if (nextVideo.enabled) {
+      widthScaleFactor += 1;
+    }
+    if (skipBackward.enabled) {
+      widthScaleFactor += 1;
+    }
+    if (skipForward.enabled) {
+      widthScaleFactor += 1;
+    }
+
+    const topOffset = Math.round((frameHeight - buttonHeight * heightScaleFactor) * 0.5);
+    const leftOffset = Math.round((frameWidth - buttonWidth * widthScaleFactor) * 0.5);
+
+    // positionStyle is for the view that acts as the container of the buttons.
+    // We want it to be centered in the player area and it is dynamic because we have different buttons
+    const positionStyle = {
+      position: 'absolute',
+      top: topOffset,
+      left: leftOffset,
+    };
+
+    if (showButton) {
+      return (
+        <View style={[positionStyle]}>
+          <Animated.View style={[containerStyle]}>
+            {previousButton}
+            {backwardButton}
+            {playPauseButton}
+            {forwardButton}
+            {nextButton}
+          </Animated.View>
+        </View>
+      );
+    }
+    return null;
+  }
+}
+
+module.exports = CastPlayPauseButtons;
```

Comment:
Use `export default class` over `module.exports`

Reply:
We will fix it during the process of lint errors fixing.